libr/core/cconfig.c

```diff
@@ -2921,6 +2921,8 @@ R_API int r_core_config_init(RCore *core) {
 	}
 	SETCB ("dir.source", "", &cb_dirsrc, "Path to find source files");
 	SETPREF ("dir.types", "/usr/include", "Default path to look for cparse type files");
+	SETPREF ("dir.home", r_sys_getenv (R_SYS_HOME), "Default path for the home directory");
```

Comment:
should have a callback that sets those env vars when the eval vars are modified, the getter should be a direct wrapper to r_sys_getenv... but thats gonna be tricky

Reply:
Sorry @radare for replying late. I am going through the other example source to see how to do what you said. If this requires an urgent fix, you can close this PR or I really want to help with this.